### ISSUE TITLE
Requests for support: Insulin Profile

### ISSUE THREAD

**Issue by Navid200** [NightscoutFoundation/xDrip#1898]
Please don't open a new issue if you want to submit a request for support for a new insulin type, or for a modification to an existing insulin profile.  
Please also don't post in this thread.  
Instead, please post in the [Discussions](https://github.com/Navid200/xDrip/discussions). Please include as much information as you can about it. We will then update this table to include your request. If there is a delay, please remind me (@Navid200).
Grouping the requests helps developers easily find the request they are looking for.  

| Insulin | Request |  
|--------|-----------|  
| Fiasp | [Mar 2018](https://github.com/NightscoutFoundation/xDrip/issues/351) |  
| Tresiba | [Jan 2020](https://github.com/NightscoutFoundation/xDrip/issues/1153) |  
| Lyumjev | [Dec 2020](https://github.com/NightscoutFoundation/xDrip/issues/1561) |  
| Humulin | [Jan 2021](https://github.com/NightscoutFoundation/xDrip/issues/1607) |

**Comment by Sufiasif** [comment]
Hi is it possible to also put request in for insulin profile humalog mix 50/50 for xdrip thanks

**Comment by Sufiasif** [comment]
Can i request insulin profile for humalog mix 50/50 on xdrip +

**Comment by rombobellogia** [comment]
I am wondering why we don't yet have a Lyumjev curve profile? is it something hard to add to xdrip?

**Comment by Arvul** [comment]
A great feature would be to add Tresiba to the NovoPen NFC sync (of course this would require Tresiba to be added) but this would increase the usability of the Novopen 6 compatibility.

**Comment by Navid200** [comment]
For those of you posting here, please note if you intentionally or unintentionally ignore the clear request not to post here, you cannot expect us to follow your requests.
The first post here contains only 6 lines telling you what to do.

**Comment by Arvul** [comment]
@Navid200 Although, there does not seem to be a discussion in the forum that concerns this issue. Would be better if the procedure was to create a topic and link directly to it instead of just writing "discuss in the forum". Especially if you want to contain the information concerning this issue.

**Comment by Navid200** [comment]
> @Navid200 Although, there does not seem to be a discussion in the forum that concerns this issue. Would be better if the procedure was to create a topic and link directly to it instead of just writing "discuss in the forum". Especially if you want to contain the information concerning this issue.

There are many insulins out there.  Do you want me to open 30 threads in the discussions in the anticipation that someone may want to request it?
I don't follow your suggestion.

If someone wants to request an insulin to be added, they can request it by opening and discussions.  Why does that not work?  Can you explain?

**Comment by Navid200** [comment]
@Arvul The request for Tresiba is already there.  Did you not see that in the table above?

**Comment by Arvul** [comment]
@Navid200 No, that's just my point: If you create one thread and link to that one directly instead of just "please post in the [Discussions](https://github.com/NightscoutFoundation/xDrip/discussions).", then you would concentrate all the discussions concerning this topic to just that one thread.

Posting in a discussion related to a closed issue is bad practice since it indicates that either the issue is of low importance or it's already fixed. Thus the linked discussion in the table seems obsolete.

**Comment by Navid200** [comment]
@Arvul I understand this may not be perfect.   

This is an open issue, not closed.  It is a summary of all the requests.
We have developers working hard on all requests and all bugs.  It's not fair to have an open issue for every single insulin xDrip does not support.

**Comment by ANDREAPIPPI** [comment]
HI. It's been a long time since the last post and nothing has happened yet.
I am one of the people who use Tresiba very long-acting bolus insulin, and I participated in the discussion on whether or not to introduce this insulin in the xDrip+ predictive trend settings.
The discussion focused, if I remember correctly, on two objections:
1- the curve is very different from all the other basals, and in the first two-three days it is different from the following days.
2- the doubt whether the basal data entered can really be of any use for the predictive trend.
As regards the first, in fact the Tresiba curve is not really a curve, or rather it is no longer one after two-three days of use, because the accumulation effect reduces the curve to an almost straight horizontal line.
So it could probably be used as such a line after that time from the beginning of its use....
As for the second, I'm a little disoriented: since yearsxDrip+ allows to insert Toujeo or Lantus long-acting bolus insulin doses, and this without any advice on their real usefulness in predicting trends, so why not Tresiba?
Seems a little inconsistent, doesn't it?
However, as a hint, Tresiba could be placed in "Engineering mode", as is the case with many other dubious settings.
And finally, why not try it? 
Perhaps we will confirm the uselessness of the insulin bolus for the predictive trend, or the opposite, may be.
By not doing, the doubt will remain unresolved
My little penny.
Andrea

**Comment by jonjonr6** [comment]
It would be great if Lyumjev [ultra-rapid lispro (URLi)] could be added to the selection list in the Xdrip+ prediction menu.
Here are links to the manufacturer's data.

PDF of profile info extracted from manufacturer's data as linked below:
[Lyumjev_profile.pdf](https://github.com/NightscoutFoundation/xDrip/files/13307503/Lyumjev_profile.pdf)

Link to manufacturer's data (see section 12 "Clinical Pharmacology":
https://uspl.lilly.com/lyumjev/lyumjev.html#pv.html#pi
